packages/core/test/unit/data-types/misc-data-types.test.ts

```diff
@@ -143,7 +143,7 @@ describe('DataTypes.JSON', () => {
       expectsql(queryGenerator.escape('string', { type: new DataTypes.JSON() }), {
         default: `'"string"'`,
         mysql: `CAST('"string"' AS JSON)`,
-        mssql: `N'"string"'`,
+        mssql: `N'string'`,
```

Comment:
This still needs to be rolled back. Escaped JSON values should be JSON strings

Reply:
Thanks for spotting this, I've updated it now